File: platform/sdk/python/examples/cloud/cloud-014_sts_token_signature_verify.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from xuanji_sdk_cloud import CloudClient

client = CloudClient({"region": "cn-east-1"})
token = "mytoken123"
signature = f"sig-{token}"
result = client.stsTokenSignatureVerify(token, signature)
if not result["valid"]:
    sys.exit(1)
print("XJ-OK: cloud-014_sts_token_signature_verify")
sys.exit(0)

File: platform/sdk/nodejs/examples/cloud/cloud-014_sts_token_signature_verify.js
const { CloudClient } = require("../../mox-sdk-cloud");
const client = new CloudClient({ region: "cn-east-1" });
const token = "mytoken123";
const signature = "sig-" + token;
const result = client.stsTokenSignatureVerify(token, signature);
if (!result.valid) process.exit(1);
console.log("XJ-OK: cloud-014_sts_token_signature_verify");
process.exit(0);
